java/libphonenumber/test/com/google/i18n/phonenumbers/PhoneNumberUtilTest.java

```diff
@@ -427,7 +427,7 @@ public void testNormaliseStripAlphaCharacters() {
 
   public void testNormaliseStripNonDiallableCharacters() {
     String inputNumber = "03*4-56&+a#234";
-    String expectedOutput = "03*456+234";
+    String expectedOutput = "03*456+#234";
```

Comment:
This should be the same as the C++ test, where you added a 1 in front of the a as well in the inputNumber.